compiler-core/src/parse/tests.rs

```diff
@@ -232,3 +232,15 @@ fn anonymous_function_labeled_arguments() {
         }
     );
 }
+
+#[test]
+fn missing_let_binding() {
+    assert_error!(
+        "let bar = 42;
```

Comment:
AFAIK the semicolons are tolerated but removed by the formatter and shouldn't be there, but I could be wrong.

Reply:
Gotcha.